### ISSUE TITLE
OCA/timesheet repo not synced to Odoo Apps?

### ISSUE THREAD

**Issue by alexey-pelykh** [OCA/maintainer-tools#482]
@sbidoul @pedrobaeza I think you two the most knowledgable on this: https://apps.odoo.com/apps/modules/browse?search=brainbean lists modules from timesheet repo only, and that's clearly outdated. All other modules (from other repos) have been updated successfully.

**Comment by sbidoul** [comment]
@alexey-pelykh this is how it looks like. The was no error displayed until I manually clicked the scan button. Maybe their auto scan feature is broken? Anyway, see the hr_timesheet_sheet dependency error...

![image](https://user-images.githubusercontent.com/692075/99145962-c4d01d80-2673-11eb-9e89-ba0f9541df04.png)

**Comment by alexey-pelykh** [comment]
@sbidoul wow, that's a bunch of errors. So it seems there's at least dependency issue and ownership issue (this one makes no sense at all)

**Comment by alexey-pelykh** [comment]
Oh, actually it's the "hr_timesheet_sheet" module that's published from other account (it seems) that's causing the issue. I believe w/o reaching out to Odoo admins it won't be fixable?

**Comment by sbidoul** [comment]
Yes someone has to reach out to apps.odoo.com admins. hr_timesheet_sheet obviously belongs to Odoo SA on older branches. I don't know if a name can be owned by different orgs on different branches.

**Comment by alexey-pelykh** [comment]
Oooooh, yeah, that could be an issue. Who of OCA has the power and mandate to outreach Odoo SA regarding this matter?

**Comment by sbidoul** [comment]
Feel free to give it a try :shrug:

**Comment by pedrobaeza** [comment]
I think the only solution is to rename our module.

**Comment by alexey-pelykh** [comment]
cc @astirpe @MiquelRForgeFlow as well

**Comment by alexey-pelykh** [comment]
@pedrobaeza I guess that's related then for all modules that OCA took over?

**Comment by pedrobaeza** [comment]
Yes. Sometimes is better that is not shown in apps (like `mrp_subcontracting` v12 backport, that we don't want to rename when migrating to v13). But for OCA/payroll, we renamed the fw-ported modules to v13, as now they follow a different path from enterprise ones.

**Comment by alexey-pelykh** [comment]
I wonder how those conflicting modules were published before 🤔 Odoo SA did not perform checks or something like that. Do we have modules that have same name as EE ones from same version?

**Comment by pedrobaeza** [comment]
I don't get your question.

**Comment by sbidoul** [comment]
> I wonder how those conflicting modules were published before

I think they were never published on apps.odoo.com. I very much doubt Odoo will change the policy regarding name ownership across branches. I think this has been discussed before.

> Do we have modules that have same name as EE ones from same version?

We should really avoid that.

**Comment by alexey-pelykh** [comment]
> I think they were never published on apps.odoo.com.

Guess you're right, since https://apps.odoo.com/apps/modules/12.0/hr_timesheet_sheet leads to nowhere and fact that https://apps.odoo.com/apps/modules/12.0/hr_timesheet_sheet_activity was somehow published before only means that apps.odoo.com did not perform check (or that was only a warning) about missing dependency?

**Comment by alexey-pelykh** [comment]
> I very much doubt Odoo will change the policy regarding name ownership across branches. I think this has been discussed before.

Yeah, that has no value to them I guess, so why bother. Thus question transforms "what OCA going to do about that and its policies" I guess. As @pedrobaeza mentioned renaming, it seems like only alternative to "doing nothing and using own app store"

**Comment by pedrobaeza** [comment]
I'm not caring at all for my modules in Odoo Apps since a lot...

**Comment by alexey-pelykh** [comment]
@pedrobaeza do you have a proper point of contact for such a talk in Odoo SA? It would be great to tag them here and see what would be the result :)

**Comment by pedrobaeza** [comment]
Nop, the interaction is through apps at odoo dot com.

**Comment by alexey-pelykh** [comment]
Well, as we assumed, they've said that our best bet is to rename our modules
